Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.ComponentModel;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_SwipeViewRenderer))]
+	public class SwipeView : ContentView, IElementConfiguration<SwipeView>
+	{
+		readonly Lazy<PlatformConfigurationRegistry<SwipeView>> _platformConfigurationRegistry;
+
+		public SwipeView()
+		{
+			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<SwipeView>>(() => new PlatformConfigurationRegistry<SwipeView>(this));
+		}
+
+		public static readonly BindableProperty LeftItemsProperty = BindableProperty.Create(nameof(LeftItems), typeof(SwipeItems), typeof(SwipeView), null, BindingMode.TwoWay, null);
```

Comment:
I don't see BindingContext being propagated

Reply:
Applied changes to add it: https://github.com/xamarin/Xamarin.Forms/blob/4355300b8ded67ae2911782b6a12a2465559917c/Xamarin.Forms.Core/SwipeItems.cs#L108